layers/meta-resin-artik/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,8 @@
+do_install_append() {
```

Comment:
I did not add a machine suffix so that this change is for both artik5 and artik10.
Question is if we add this change for both of these machines at the same time or not. If so, there's no need for suffix. Right?